Add ReactInitializer section (#366)
microsoft/react-native-windows-samples: docs/native-modules-advanced.md

});
```

## Native Modules with Initializer and as a way to access ReactContext

If your native module needs to perform some initialization logic on the native (Csharp) side, there is an easy mechanism for you to do so when the app is setting up. ALl you need to do is add a method that takes a [`ReactContext`](IReactContext) and has `[ReactInitializer]` attribute.

> An example could be if you need to do something periodically on the native side for your module, setting up the timer during initialization makes sense.

Reviewer: ```suggestion
If your native module needs to perform some operation periodically, you can do so by setting up a timer during your module's initialization as in the following example:
```
